src/components/addBookForm.jsx

```diff
@@ -32,10 +32,13 @@ const AddBookForm = () => {
   };
 
   return (
-    <form action="" onSubmit={(e) => handleSubmit(e)}>
-      <input type="text" placeholder="Title" value={title} onChange={(e) => setTitle(e.target.value)} required />
-      <input type="text" placeholder="Author" value={author} onChange={(e) => setAuthor(e.target.value)} required />
-      <button type="submit">Add book</button>
+    <form action="" onSubmit={(e) => handleSubmit(e)} className="add-book-form">
+      <h2 className="form-header">Add new book</h2>
+      <div className="inputs-container">
+        <input type="text" placeholder="Book title" className="title-input" value={title} onChange={(e) => setTitle(e.target.value)} required />
+        <input type="text" placeholder="Author" className="author-input" value={author} onChange={(e) => setAuthor(e.target.value)} required />
```

Comment:
- [ ] [OPTIONAL]
Kindly note that for the form, the original design has inputs for Book title and _Category_, instead of _Author_. Let's also improve. Let's also improve upon width of the form, it should match the width of the Book cards.